docs/admin/multiple-schedulers.md

```diff
@@ -0,0 +1,214 @@
+
+
+1. Pod spec without any scheduler annotation
+
+  ```yaml
+  apiVersion: v1
+  kind: Pod
+  metadata:
+    name: no-annotation
+    labels:
+      name: multischeduler-example
+  spec:
+    containers:
+    - name: pod-with-no-annotation-container
+      image: gcr.io/google_containers/pause:2.0
+  ```
+
+  When no scheduler annotation is supplied, the pod is automatically scheduled using the default-scheduler.
+
+  Save this file as `pod1.yaml` and submit it to the Kubernetes cluster.
+
+  ```sh
+  $ kubectl create -f pod1.yaml
+  ```
+2. Pod spec with `default-scheduler` annotation
+
+  ```yaml
+  apiVersion: v1
+  kind: Pod
+  metadata:
+    name: annotation-default-scheduler
+    annotations:
+      scheduler.alpha.kubernetes.io/name: default-scheduler
+    labels:
+      name: multischeduler-example
+  spec:
+    containers:
+    - name: pod-with-default-annotation-container
+      image: gcr.io/google_containers/pause:2.0
+  ```
+
+  A scheduler is specified by supplying the scheduler name as a value to the annotation key `scheduler.alpha.kubernetes.io/name`. In this case, we supply the name of the default scheduler which is `default-scheduler`.
+
+  Save this file as `pod2.yaml` and submit it to the Kubernetes cluster.
+
+  ```sh
+  $ kubectl create -f pod2.yaml
+  ```
+3. Pod spec with `my-scheduler` annotation
+
+  ```yaml
+  apiVersion: v1
+  kind: Pod
+  metadata:
+    name: annotation-second-scheduler
+    annotations:
+      scheduler.alpha.kubernetes.io/name: my-scheduler
+    labels:
+      name: multischeduler-example
+  spec:
+    containers:
+    - name: pod-with-second-annotation-container
+      image: gcr.io/google_containers/pause:2.0
+  ```
+
+  In this case, we specify that this pod should be scheduled using the scheduler that we deployed - `my-scheduler`. Note that the scheduler name in the annotation key should match the name supplied to the scheduler command as an argument in the deployment config for the scheduler.
+
+  Save this file as `pod3.yaml` and submit it to the Kubernetes cluster.
+
+  ```sh
+  $ kubectl create -f pod3.yaml
+  ```
+
+  Verify that all the pods are running.
+
+  ```sh
+  $ kubectl get pods
+  ```
+
+### Verifying that the pods were scheduled using the desired schedulers
+
+In order to make it easier to work through these examples, we did not verify that the pods were actually scheduled using the desired schedulers. We can verify that by changing the order of pod and deployment config submissions above. If we submit all the pod configs to a Kubernetes cluster before submitting the scheduler deployment config, we see that the pod `annotation-second-scheduler` remains in "Pending" state for ever while the other two pods get scheduled. Once we submit the scheduler deployment config, `annotation-second-scheduler` pod gets scheduled as well.
```

Comment:
s/Once we submit the scheduler deployment config, `annotation-second-scheduler` pod/Once we submit the scheduler deployment config and our new scheduler starts running, the `annotation-second-scheduler` pod/